### ISSUE TITLE
Add an option to "autofix" file on saving

### ISSUE THREAD

**Issue by BLaDZer** [joakim-eriksson/nb-eslint#39]
I think it's very usefull option that already exist in Phpstorm and VScode extension. Also I'm using same function in another plugin called "PHP CS Fixer" https://plugins.netbeans.apache.org/catalogue/?id=36

**Comment by Chris2011** [comment]
Yes, good point. Thx for the info. I will definitely implement it, but my time is limited unfortunately. But it is planned :).
